lua/lualine/components/buffers/init.lua

```diff
@@ -77,7 +83,22 @@ function M:buffers()
   local buffers = {}
   M.bufpos2nr = {}
   for b = 1, vim.fn.bufnr('$') do
-    if vim.fn.buflisted(b) ~= 0 and vim.api.nvim_buf_get_option(b, 'buftype') ~= 'quickfix' then
+    local valid = vim.api.nvim_buf_is_valid(b)
+    local ft
+    if valid then
+      ft = vim.api.nvim_buf_get_option(b, 'filetype')
+    end
+    -- don't want the user cyclying into this buffer if they've listed it as an excluded filetype in their config
+    -- go ahead and mark it unlisted automatically
+    if valid and self.excludedFT[ft] then
+        vim.api.nvim_buf_set_option(b, 'buflisted', false)
+    end
+    local cond = vim.fn.buflisted(b) ~= 0
+                 and vim.api.nvim_buf_get_option(b, 'buftype') ~= 'quickfix'
+                 and self.excludedFT[ft] == nil
+                 and valid
+    if cond then
```

Comment:
I think it'd be better if you put the condition in the if block instead of in another variable to be consistent with rest og the code base